v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_collection.py

```diff
@@ -1,19 +1,28 @@
+import logging
+from typing import ClassVar
+
 import hail as hl
 import luigi
 
 from v03_pipeline.lib.model import ReferenceDatasetCollection
 from v03_pipeline.lib.paths import valid_reference_dataset_collection_path
 from v03_pipeline.lib.reference_data.dataset_table_operations import (
     update_or_create_joined_ht,
+    validate_joined_ht_globals_match_config,
 )
 from v03_pipeline.lib.tasks.base.base_update_task import BaseUpdateTask
 from v03_pipeline.lib.tasks.files import GCSorLocalTarget
 
+logger = logging.getLogger(__name__)
+
 
 class UpdatedReferenceDatasetCollectionTask(BaseUpdateTask):
     reference_dataset_collection = luigi.EnumParameter(enum=ReferenceDatasetCollection)
     dataset = luigi.OptionalStrParameter(default=None)
```

Comment:
We don't even need `dataset` anymore here do we?

Reply:
removed `dataset` task parameter